includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -3303,7 +3303,9 @@ static function ( $attribute_name ) {
 					return sprintf(
 						/* translators: %1$s is the invalid attribute value, %2$s is the attribute name */
 						esc_html__( 'Invalid value %1$s for attribute %2$s', 'amp' ),
-						'<code>' . esc_html( $validation_error['node_attributes'][ $validation_error['attribute'] ] ) . '</code>',
+						empty( $validation_error['node_attributes'][ $validation_error['attribute'] ] )
+							? '(' . esc_html__( 'empty', 'amp' ) . ')'
```

Comment:
IMO, the parentheses would make sense to include as part of the translated string, but I don't feel strongly about it.

Reply:
Yes, make sense to include it. Translators will automatically handle this case IMO.